<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputDialog</class>
 <widget class="QDialog" name="OutputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Face Recognition Access System</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icon/icon.png</normaloff>:/icon/icon.png</iconset>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Access">
    <attribute name="title">
     <string>Access</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>90</y>
       <width>291</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Details</string>
     </property>
     <widget class="QLabel" name="StatusLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto Condensed Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="NameLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>169</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto Condensed Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto Condensed Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>54</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto Condensed Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Name : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>107</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto Condensed Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Clocked Time : </string>
      </property>
     </widget>
     <widget class="QLabel" name="HoursLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto Condensed Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="imgLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>471</width>
       <height>381</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>4</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="visitorButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>290</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Visitor Access</string>
     </property>
    </widget>
    <widget class="QPushButton" name="residentBtn">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>290</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Resident Access </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>50</y>
       <width>62</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Condensed Light</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Time :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Time_Label">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>50</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>59</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Condensed Light</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Date :</string>
     </property>
    </widget>
    <widget class="QLabel" name="Date_Label">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="adminButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>350</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Admin</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
